google/cloud/security/inventory/inventory_loader.py

```diff
@@ -17,11 +17,14 @@
 Usage:
 
   $ forseti_inventory \\
+      --inventory_groups \\
+      --service_account_email <email of the service account> \\
```

Comment:
I wanted to keep the PRs very focused. A document update is being prepared [here](https://github.com/GoogleCloudPlatform/forseti-security/compare/groups-docs)